Wrappers/Python/cil/recon/FBP.py

```diff
@@ -319,7 +359,7 @@ def _calculate_weights(self, acquisition_geometry):
         (yy, xx) = np.meshgrid(xv, yv)
 
         principal_ray_length = ag.dist_source_center + ag.dist_center_detector
-        scaling =  ag.magnification * (2 * np.pi/ ag.num_projections) / ( 4 * ag.pixel_size_h ) 
+        scaling = 0.25 * ag.magnification * (2 * np.pi/ ag.num_projections) / ag.pixel_size_h
```

Comment:
why this (non) change?

Reply:
because  I was trying to figure out where these values came from due to TIGRE's implementation and it's a bit easier to read.